docs/src/style/index.md

```diff
@@ -156,13 +156,27 @@ This can be through inheriting from it or by directly copying artifacts.
 The main point is that a subproject should not have multiple builds scattered in various places.
 Each subproject has an authoritative `build` target that *all* targets use when fetching build artifacts.
 
-### Prefer User Defined Commands (UDCs)
-
-The primary purpose of a UDC is to reduce boilerplate and promote reusing common workflows.
-Many build patterns tend to be repetitive.
-For example, copying a package lockfile and installing dependencies is very common.
-
-In these cases, a UDC should be preferred.
-The `catalyst-ci` repository provides a number of UDCs in the `earthly` subdirectory.
-These should be used prior to writing a new one.
-If a common use case is not covered in this subdirectory, a PR should be opened to add it.
+### Earthly User Defined Commands (UDCs) Deprecation Starting from Earthly 0.8
+
+Please be advised that Earthly's File Descriptor Cache (FDC) is deprecated, effective upon updating to Earthly 0.8.
+Update your configurations accordingly to ensure a seamless transition and refer to the documentation for guidance.
+
+### Use Functions instead of User Defined Commands (UDCs) in Earthly 0.8
```

Comment:
We can just make it straight to the point. Maybe just use the old document

### Prefer `FUNCTION`
- The primary purpose of a UDC is to reduce boilerplate and promote reusing common workflows.
- Many build patterns tend to be repetitive.
- For example, copying a package lockfile and installing dependencies is very common.
- In these cases, a `FUNCTION` should be preferred.
- The `catalyst-ci` repository provides a number of `FUNCTION`s in the `earthly` subdirectory.
- These should be used prior to writing a new one.
- If a common use case is not covered in this subdirectory, a PR should be opened to add it.